automation/taskcluster/l10n/locales.py

```diff
@@ -42,7 +42,7 @@ def get_release_locales():
     for line in file:
         if line == OPEN_LOCALES:
             locales_opened = True
-        elif line == CLOSE_LOCALES:
+        elif line == CLOSE_LOCALES and locales_opened == True:
```

Comment:
Does this catch an edge case?

Reply:
It's so we don't exit the loop when we see the closing bracket for the regular `locales` list.